.dockerignore

```diff
@@ -0,0 +1,29 @@
+*.pyc
+include/
+lib/
+staticfiles/
+media/
+.build
+
+# sqlite3 db
+*.db
+*.sqlite3
+
+# sublime stuff
+*.sublime-workspace
+*.sublime-project
```

Comment:
I'm not sure about there being a global .dockerignore (https://github.com/docker/docker/issues/12843) for .gitignore I agree